### DISCUSSION TITLE
Petalinux support for KR260

### DISCUSSION THREAD

**Discussion by slai-mark** [corundum/corundum#182] [Q&A] [answered]
Is there any documentation for how to build the petalinux image for the KR260 please?

**Comment by alexforencich** [comment, marked as answer]
At the moment, no. Personally, I prefer using the Ubuntu images as they're much easier to deal with, at least on the Xilinx dev boards. The KR260 also boots differently vs. other boards, I have not had time to figure out how to set up petalinux to boot on that board.

**Reply by slai-mark** [reply]
Hi, thanks for the response.  Yes, the KR260 is a little quirky compared to the other Xilinx dev boards we're using.
In terms of the petalinux code, as a bare minimum is it just modules/mqnic and utils which are required?
